modules/mono/csharp_script.h

```diff
@@ -188,6 +188,13 @@ class CSharpScript : public Script {
 	String get_source_code() const override;
 	void set_source_code(const String &p_code) override;
 
+#ifdef TOOLS_ENABLED
+	virtual const Vector<DocData::ClassDoc> &get_documentation() const override {
+		static Vector<DocData::ClassDoc> docs;
+		return docs;
```

Comment:
We can definitely support this in C# in the future. Could you add a `// TODO` here?

Reply:
@neikeq Okey!